Timezones/Timezones.plugin.js

```diff
@@ -241,7 +241,7 @@ module.exports = (() => {
                               return `${hours.toString().length === 1 ? `0${hours}` : hours}:${minutes.toString().length === 1 ? `0${minutes}` : minutes}`;
                           }
 
-                          const hour = hours > 12 ? hours - 12 : hours;
+                          const hours = hours > 12 ? hours - 12 : (hours == 0 ? 12 : hours);
```

Comment:
Why are you changing `hour` to `hours`?

Reply:
honestly i don't know at this point
i just changed it back :)